src/command.h

```diff
@@ -183,6 +187,52 @@ class DrawRectCommand : public PipelineCommand {
   float height_ = 0.0;
 };
 
+/// Command to draw a grid of recrangles on screen.
+class DrawGridCommand : public PipelineCommand {
+ public:
+  explicit DrawGridCommand(Pipeline* pipeline, PipelineData data);
+  ~DrawGridCommand() override;
+
+  const PipelineData* GetPipelineData() const { return &data_; }
+
+  void EnableOrtho() { is_ortho_ = true; }
```

Comment:
Ortho is only ever true, can we remove the member and just set it as needed?

Reply:
Copied off draw_rect. Removed here, left in draw_rect.